test/data.jl

```diff
@@ -305,6 +305,20 @@ module TestData
     # @test (3 in unique(dv))
     # @test (NA in unique(dv))
 
+    # test nonunique() with extra argument
+    df = DataFrame(a = ["a", "b", "a", "b", "a", "b"], b = 1:6, c = [1:3;1:3])
+    df = vcat(df, df)
+    @test isequal(sum(nonunique(df)), 6)
```

Comment:
Use `==` instead of `isequal` unless there's a specific reason not to (e.g. `NaN`).

Could you test the actual contents of the data frame instead? That way, we're completely sure the code will continue to work, and not return garbage.